test.js

```diff
@@ -7,4 +7,6 @@ test('main', t => {
 	t.deepEqual(splitOnFirst('a---b---c', '---'), ['a', 'b---c']);
 	t.deepEqual(splitOnFirst('a-b-c', '+'), ['a-b-c']);
 	t.deepEqual(splitOnFirst('abc', ''), ['abc']);
+	const error = t.throws(() => splitOnFirst('abc', null), TypeError);
```

Comment:
You can test both the error type and message directly in the throws assertion.

Reply:
I was try use:
```js
t.throws(() => splitOnFirst('abc', null), TypeError, 'any string')
```

but it passed

Reply:
That is not correct. Read the `t.throws` docs. It's all there.

Reply:
Thanks, I was fixed it